### ISSUE TITLE
Possible tiny-lr API change?

### ISSUE THREAD

**Issue by krulik** [bestander/chrome-app-livereload#2]
Gulp complains there's no method `listen`.

Also the path to the `livereload.js` file had changed.

Can you update the post/repo?

Thanks!

**Comment by krulik** [comment]
Sorry, my mistake, I've missed the `var lr = tinylr();` piece in your code :)

The path did change though, and I'm still having problems making it work, currently it doesn't reload the page. I've made the `chrome.runtime` change (in the `/dist/livereload.js` file, there's also a `/src` one).

UPDATE: I've discovered that the `this.window.chrome.runtime.reload()` throws an error because the `runtime` object doesn't have a `reload` method. I'm running an extension. Checking wether there's a `sendMessage` equivalent.

UPDATE: Well for some unknown reason I don't have the `reload` method available in the `chrome.runtime` object.

UPDATE: I've desperately tried reverting the `window.location` back and it worked :) If it helps anyone...
